### PR TITLE
Adding Keep project

### PR DESCRIPTION
**Pull request by talboren**
👋 I think my project (https://www.github.com/keephq/keep) would be a great fit for Up for Grabs.

Let me know if there's any feedback you have about the template :)

Thanks!

### CODE REVIEW

**Comment by talboren** [comment]
@shiftkey can you elaborate on what work is needed here please?

**Comment by shiftkey** [comment]
@talboren please see the comment from the review automation:

> I couldn't find the label that was used in the `upforgrabs.link` value: 'good first issues'. Please update the configuration to match the right label in this repository.

There doesn't seem to be a label in your repository that matches this value

File: _data/projects/keep.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+desc: Open-source alert management and workflow automation platform.
+site: https://www.keephq.dev
+tags:
+  - monitoring
+  - alerting
+  - alerts
+  - workflow-automation
+  - automation
+upforgrabs:
+  name: good first issues
```
Comment (shiftkey) [inline _data/projects/keep.yml]: I think this is the problem - the `s` on the end is not actually part of the label

```suggestion
  name: good first issue
```

File: _data/projects/keep.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  link: https://github.com/keephq/keep/issues?q=label%3A%22good%20first%20issue%22
```
Comment (shiftkey) [inline _data/projects/keep.yml:12]: :art: a simpler URL that's equivalent to the current value

```suggestion
  link: https://github.com/keephq/keep/labels/good%20first%20issue
```

Comment (talboren) [reply _data/projects/keep.yml:12]: Gotcha, thanks!

**Comment by talboren** [comment]
@shiftkey i think it’s ok now :)

**Review by shiftkey — APPROVED** [review]

**Pull request merged**